Tom-tom7m/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -0,0 +1,31 @@
+/*   Write a script that prints the lyrics to "99 Bottles of Beer on the Wall"
+in the terminal.    If you're unfamiliar with the song, you can [find the
+lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+  
+  Be sure that all of your output includes proper spacing (no words or values
+  running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string
+  \n to insert a line break.
+  
+  For an extra challenge, improve this code to print "1 bottle" rather than "1
+bottles." (Hint: read up on if/else   conditional statements at
+https://developer.mozilla.org/en-
+US/docs/Web/JavaScript/Reference/Statements/if...else).  */
+
+ "use strict";
+
+let bottle, bottleNum;
+for(bottleNum = 99; bottleNum > 0; bottleNum --){
+	if(bottleNum == 1){
```

Comment:
One thing to note is it's also generally best practice to try and use `===` instead of `==` when comparing for equality, this makes sure the actual values are equal and doesn't cause problems by automatic type conversion.